<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="execute_button">
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="query_text"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="query_history"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
